docs/pages/tutorials/integrating-with-astro.md

```diff
@@ -0,0 +1,108 @@
+---
+meta:
+  title: Integrating with Astro
+  description: This page explains how to integrate Shoelace with an Astro app.
+---
+
+# Integrating with Astro
+
+This page explains how to integrate Shoelace with an Astro app.
+
+:::tip
+This is a community-maintained document. Please [ask the community](/resources/community) if you have questions about this integration. You can also [suggest improvements](https://github.com/shoelace-style/shoelace/blob/next/docs/tutorials/integrating-with-astro.md) to make it better.
+:::
+
+## Instructions - Astro 4.11.3
+
+- Node: v18.17.1 or v20.3.0 or higher. ( v19 is not supported.)
+- Astro: 4.11.3
+- Shoelace: 2.15.1
+
+To get started using Shoelace with Astro, the following packages must be installed.
+
+```bash
+npm install @shoelace-style/shoelace
```

Comment:
```suggestion
npm install @shoelace-style/shoelace rollup-plugin-copy
```

Reply:
Just curious, what is that doing exactly? I don't have it in my setup.